PAMPHLET.md

```diff
@@ -0,0 +1,71 @@
+# Doorboard Web Application
+
+The power of **Doorboard** is the ability to share messages that were traditionally on a post-it note in an online format for greater access.
+
+**Doorboard** is a lightweight web application that was designed to allow members within an organization to quickly communicate updates about their availability during scheduled hours. It is meant to replace the iconic “post-it” note and improve upon it to provide a more meaningful medium of communication. Members within an organization no longer need to worry that their message is not communicated to those that are not local to their facility (e.g. remote learners, employees, affiliates, etc.). If things should change, members can quickly access the platform to manage their notes to keep information current. **Doorboard** can be accessed via a normal computer browser or with mobile devices.
+
+**Doorboard** uses the nomenclature **Note**, **Owner** and **Viewer** to differentiate between member posts (posted messages), member users (**Doorboard** owners) and the end users for that specific organization.
+
+**Doorboard** is split into two main views, each with their own functionality:
+- **Owner Page** - member view for managing notes
+  ![owner page](doorboard_pamphlet_images/Doorboard%20Owner%20Page.png)
+- **Viewer Page** - end user view for accessing member notes (read-only access) and their google calendar
+  ![viewer page](doorboard_pamphlet_images/Doorboard%20Viewer%20Page.png)
+  ![viewer page gcal button](doorboard_pamphlet_images/Doorboard%20g-cal%20button.png)
+  ![viewer page gcal](doorboard_pamphlet_images/Doorboard%20-%20Gcal%20view.png)
+
+Within each view, **Notes** are displayed with the following information:
+- **Timestamps** - time of creation or last modification
+- **Body** - text of the note
+- **Pin Star & Background Color** - indicates important messages that have been highlighted by the **Doorboard** member
+  ![note pinning & background](doorboard_pamphlet_images/Doorboard%20Live%20&%20Pinned%20Notes.png)
+
+The **Owner Page** provides the interface for members to manage their "live" and "trashed" Notes. The main features are as follows:
+- **Authentication** - authentication is managed through google's Auth0 platform
+  <img src="/doorboard_pamphlet_images/Doorboard%20Auth0%20Login.png" alt="Authentication" width = "450" height = "620"/>
```

Comment:
The images you did in html look squished. Maybe switch to markdown?

Reply:
They look awful if I try to use native markdown since it doesn't have the capability of resizing. Should I try a different size for better apperance?

Reply:
Yeah, definitely try something else